package.json

```diff
@@ -39,19 +40,22 @@
     "multer": "^1.0.1",
     "newrelic": "1.5.5",
     "nunjucks": "1.3.4",
+    "properties-parser": "^0.3.1",
```

Comment:
We might want to use greenkeeper to get updates, so removing the caret is better here.